### DISCUSSION TITLE
New DistroAV Problem

### DISCUSSION THREAD

**Discussion by EgyptianSky** [DistroAV/DistroAV#1145] [Q&A - Support]
So I have been trying to get distroAv to work but it popped up with this on obs what is the next step for me to fix this?
<img width="1470" alt="Screenshot 2024-11-17 at 5 05 03 PM" src="https://github.com/user-attachments/assets/8abe81ca-784d-47e9-bc88-a6d84a623e5a">

**Comment by RayneYoruka** [comment]
Please read the install instructions. https://github.com/DistroAV/DistroAV/wiki/1.-Installation

**Reply by EgyptianSky** [reply]
Got that part but I am still hitting a hiccup. I got the obs link to pop up but no gameplay.
<img width="1470" alt="Screenshot 2024-11-17 at 5 57 10 PM" src="https://github.com/user-attachments/assets/cd4e27bc-9dbb-4c8b-8f4f-c184d49ec205">
Second is that NDI Pluggin supposed to have that caution symbol?
<img width="1470" alt="Screenshot 2024-11-17 at 5 57 24 PM" src="https://github.com/user-attachments/assets/e682d37b-606f-487f-919e-6d665a00736b">

**Comment by RayneYoruka** [comment]
Wasn't obs link discontinued? The project has changed name from obs-ndi to DistroAV. The paths ans everything have changed so older programs need to update to supoort it or you're stuck using an old version of this plugin.

**Reply by EgyptianSky** [reply]
ok so no more obs link gotcha. so have downloaded distroav and the runtime. I have uninstalled ndi and it should just pop up on obs ?

**Comment by RayneYoruka** [comment]
It will as long as you follow install instructions
